skyportal/handlers/api/internal/util.py

```diff
@@ -0,0 +1,153 @@
+import datetime
```

Comment:
As per my comment above, I'd suggest moving this file to `skyportal/offset_util.py`